roles/openshift_health_checker/openshift_checks/sdn.py

```diff
@@ -0,0 +1,354 @@
+                                         ['/bin/docker', 'version'])
+                self.save_command_output('oc-version', ['/bin/oc', 'version'])
+                self.register_file('os-version', None,
+                                   '/etc/system-release-cpe')
+            except OpenShiftCheckException as exc:
+                self.register_failure(exc)
+
+        group_names = self.get_var('group_names', default=[])
+        if 'oo_masters_to_config' in group_names:
+            self.check_master()
+        if 'oo_nodes_to_config' in group_names:
+            self.check_node()
+        return {}
+
+    def save_command_output(self, path, command):
+        """Execute the provided command using the command module
+        and save its output to the specified file.
+
+        If the command is a string, use a shell.  Otherwise, assume the command
+        is a list, join it with spaces, and execute it without shell.
+        """
+        self.register_file(path, self.read_command_output(command))
+
+    def read_command_output(self, command):
+        """Execute the provided command using the command module
+        and return its output.
+
+        If the command is a string, use a shell.  Otherwise, assume the command
+        is a list, join it with spaces, and execute it without shell.
+        """
+        uses_shell = False
+        if isinstance(command, six.string_types):
+            uses_shell = True
+        else:
+            command = ' '.join(command)
+
+        command_args = dict(_raw_params=command, _uses_shell=uses_shell)
+        # Use self._execute_module instead of self.execute_module because
+        # the latter sets self.changed.
+        result = self._execute_module('command', command_args)
+        if result.get('rc', 0) != 0 or result.get('failed'):
+            raise OpenShiftCheckException(
+                'RemoteCommandFailure',
+                'Failed to execute command on remote host: %s' % command)
+
+        return result['stdout'].encode('utf-8')
+
+    def check_master(self):
+        """Gather diagnostic information on a master and ensure it can connect
+        to kubelets."""
+        if self.want_full_results:
+            conf_base_path = self.get_var('openshift.common.config_base')
+            master_conf_path = os.path.join(conf_base_path, 'master',
+                                            'master-config.yaml')
+            self.register_file('master-config.yaml', None, master_conf_path)
+
+            self.save_component_container_logs('controllers', 'controllers')
+            self.save_component_container_logs('api', 'api')
+
+        nodes = self.get_resource('nodes')
+        pods = self.get_resource('pods')
+        services = self.get_resource('services')
+
+        if self.want_full_results:
+            self.register_file('nodes.json', nodes)
+            self.register_file('pods.json', pods)
+            self.register_file('services.json', services)
```

Comment:
self.register_file('pods.json',  self.get_resource('pods'))
self.register_file('services.json', self.get_resource('services'))

pod and services in L116-117 is not used outside of L119 condition.

Reply:
Fixed.